keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -627,6 +682,8 @@ details). For now, we leave it here.
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+No new API calls are introduced as part of this feature.
+
```

Comment:
some hints on "What steps should be taken if SLOs are not being met to determine the problem?" would be good.  Likely check the provisioner logs on the node in question?

Reply:
Done!